feat: style TWAP warning banner

fairlighteth opened the pull request:
# Summary

<img width="1580" alt="Screenshot 2024-12-13 at 15 05 14" src="https://github.com/user-attachments/assets/1f8dc606-c21a-40ae-b829-a9fd83d8a5ab" />
<img width="1651" alt="Screenshot 2024-12-13 at 15 05 08" src="https://github.com/user-attachments/assets/15b435a5-2cdb-441e-a2eb-5a0177abc6ec" />

elena-zh reviewed (commented):
Hey @fairlighteth , great!
However, is this OK that we have a bit different text for the fallback handler on 2 related warnings which can be displayed at the same time? 
![Screenshot_2](https://github.com/user-attachments/assets/c9cce704-6de9-4bf4-90e6-4546cdf90048)

![Screenshot_1](https://github.com/user-attachments/assets/3aafa93d-316b-4225-82c2-5ca9b72dadb9)

I'm not sure here, would it be possible to have the same style for the warning icon? At the top, it is displayed from the left. At the bottom, it is center-aligned. 

Also, there is a typo in the text, could you please fix it? 
![image](https://github.com/user-attachments/assets/0dd71f6b-36b7-4e0e-9fee-0f143e69163f)

fairlighteth commented:
@elena-zh pushed some changes:
- Fixed typo
- Changed banner orientation

As for the other item, I'd say out of scope for this particular PR. IMO we need to assess this separately.


<img width="921" alt="Screenshot 2024-12-16 at 12 52 33" src="https://github.com/user-attachments/assets/ec75c04c-d6ee-4236-8cad-62af22c7fa54" />
<img width="1553" alt="Screenshot 2024-12-16 at 12 52 24" src="https://github.com/user-attachments/assets/32967ee0-9edf-43fa-a3b8-06bb53ae56e6" />
